ci/jenkins.md

```diff
@@ -10,6 +10,11 @@ docker run \
 jenkins/jenkins:2.262-jdk11
 ```
 
+if You use Windows command line, input like this: (You have install Docker, of course)
+```shell
+docker run -p 8080:8080 -p 50000:50000 jenkins/jenkins:2.262-jdk11
```

Comment:
Is line `-v ~/jenkins_local:/var/jenkins_home` skipped intentionally?

Reply:
yep, windows does not want to work with it:))